src/main/webapp/WEB-INF/jsp/access_confirmation.jsp

```diff
@@ -11,7 +11,7 @@
   <title>OpenClinica</title>
   <link type="text/css" rel="stylesheet" href="<c:url value="/style.css"/>"/>
   <script type="text/JavaScript" language="JavaScript" src="../includes/jmesa/jquery.min.js"></script>
-  <script type="text/javascript" language="JavaScript" src="../includes/jmesa/jquery-migrate-1.4.1.js"></script>
+  <script type="text/javascript" language="JavaScript" src="../includes/jmesa/jquery-migrate-3.1.0.min.js"></script>
```

Comment:
can these scripts be moved to a more common place (like navBar.jsp), so you won't have to reference them in each jsp file?

Reply:
OK moved